rootfs/usr/local/bin/gen-dkim

```diff
@@ -4,14 +4,17 @@
 DKIM_PRIVATE_KEY=/data/dkim/${ANONADDY_DOMAIN}.private
 
 if [ -z "$ANONADDY_DOMAIN" ]; then
-  >&2 echo "ERROR: ANONADDY_DOMAIN must be defined"
+  echo >&2 "ERROR: ANONADDY_DOMAIN must be defined"
```

Comment:
my editor has shell linting (shfmt + shellcheck) so this was auto applied. i can revert if desired.